tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_pin_page.py

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+from gaiatest.apps.search.app import Search
+
+
+class TestBrowserPinPage(GaiaTestCase):
+
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+
+        self.test_url = self.marionette.absolute_url('mozilla.html')
+
+
+    def test_browser_pin_page(self):
+        search = Search(self.marionette)
+        search.launch()
+
+        browser = search.go_to_url(self.test_url)
+        browser.tap_menu_button()
+        pin_page = browser.tap_pin_button()
+
+        self.marionette.switch_to_frame()
+
+        pin_page.tap_pin_page_to_home_screen()
+
+        # Switch to Home Screen to look for Pin site
+        self.device.touch_home_button()
+
+        homescreen = Homescreen(self.marionette)
+
+        homescreen.swipe_screen()
+
+        # Recover the last icon listed in the HomeScreen
+        last_icon_position = len(homescreen.app_elements) - 1
+        last_icon = homescreen.app_elements[last_icon_position]
+
+        # Check Data-identifier
```

Comment:
I'd remove that comment too.

`data-identifier` is a detail that is not shown in the test. If one day, the homescreen changes and `data-identifier` is replaced by `url`, we'll have to go back in the test to change that comment. That might be something that people forget. So this comment will end up lying.